src/SFA.DAS.Reservations.Web/Controllers/ReservationsController.cs

```diff
@@ -82,17 +169,17 @@ public async Task<IActionResult> Create(string employerAccountId, int? ukPrn, st
                     ModelState.AddModelError(member.Split('|')[0], member.Split('|')[1]);
                 }
 
-                var model = await BuildApprenticeshipTrainingViewModel(ukPrn);
+                var model = await BuildApprenticeshipTrainingViewModel(1234);
```

Comment:
Does this want to be that value?

Reply:
it will definitely work if your ukprn is 1234.